sshuttle/client.py

```diff
@@ -63,12 +64,14 @@ def got_signal(signum, frame):
 ACL_EXCLUDED_SOURCES_TYPE = 4
 ALLOWED_UDP_ACL_TYPE = 5
 ALWAYS_CONNECTED_TYPE = 6
```

Comment:
Do we still need ALWAYS_CONNECTED_TYPE since we have ACL_ALWAYS_CONNECTED_TYPE now?

Reply:
yes, we still check both to determine if the workspace is allowed. The room feature is ALWAYS_CONNECTED_TYPE which must be turned ON and the list of workspaces that are enabled is ACL_ALWAYS_CONNECTED_TYPE

Reply:
added comments to explain